docs/columns.md

```diff
@@ -64,6 +64,50 @@ The following example uses the file [sample.tsv](https://raw.githubusercontent.c
 2. Press `s` or `t` to select the rows referencing the columns you wish to type.
 3. Type `g` followed by the any of the above typing keystrokes to set the type for all selected columns on the source sheet.
 
+---
+
+## How to format numeric columns to contain a thousands separator within a session?
+
+**Note**: tsvs and csvs save as they are displayed.
+
+1. Set a column to a numeric type by pressing `#` (int), `%` (float), or `$` (currency).
+2. Press `C` to open the **Columns Sheet**.
+3. Move to the row referencing the column whose display you wish to format. Move the cursor to the fmtstr column.
+4. Type `e` followed by *{:,.0f}* for an `int` type and *{:,.02f}* for a floating point type.
+
+This changes the formatting for a specific column.
+
+or
+
+1. Press `O` to open the **Options Sheet**.
+2. Move the cursor down to the relevant *disp_TYPE_fmt* option where *TYPE* can be *int*, *currency*, or *float*.
```

Comment:
or `date`.

Reply:
Intentionally excluded because this was very specifically about thousands separator, but since we are generalising it, I will include it!